metadata-io/src/test/java/com/linkedin/metadata/system_info/SystemInfoServiceTest.java

```diff
@@ -0,0 +1,277 @@
+package com.linkedin.metadata.system_info;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+import static org.testng.Assert.*;
+
+import com.linkedin.metadata.system_info.collectors.PropertiesCollector;
+import com.linkedin.metadata.system_info.collectors.SpringComponentsCollector;
+import java.util.Map;
+import java.util.concurrent.ExecutorService;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class SystemInfoServiceTest {
```

Comment:
I'm not a huge fan of these unit tests, the classes being tested are trivial and these tests only add to the bulk. I added the tests because Codecov was complaining.